one_shot_scripts/update_draft_format.js

```diff
@@ -0,0 +1,120 @@
+const mongoose = require('mongoose');
+const Draft = require('../models/draft');
+const mongosecrets = require('../../cubecobrasecrets/mongodb');
+var carddb = require('../serverjs/cards.js');
+
+const batch_size = 100;
+
+async function update(draft) {
+  draft.seats = [];
+  draft.unopenedPacks = [];
+
+  //add player
+  const playerSeat = {
+    bot: null,
+    userid: draft.owner,
+    name: draft.username,
+    pickorder: draft.pickOrder,
+    drafted: draft.picks[0],
+    packbacklog: draft.packs[0] && draft.packs[0][0] ? [draft.packs[0][0]] : [],
+  };
+
+  draft.seats.push(playerSeat);
+  draft.unopenedPacks.push(draft.packs[0] ? draft.packs[0].slice(1) : []);
+
+  //add bots
+  for (let i = 1; i < draft.picks.length; i++) {
+    const bot = {
+      bot: draft.bots[i - 1],
+      name: 'Bot ' + i + ': ' + draft.bots[i - 1][0] + ', ' + draft.bots[i - 1][1],
+      pickorder: draft.picks[i],
+      drafted: [],
+      packbacklog: draft.packs[i] && draft.packs[i][0] ? [draft.packs[i][0]] : [],
+    };
+
+    //now we need to build picks from the pickorder ids
+    for (let j = 0; j < 16; j++) {
```

Comment:
What is this number from (15 picks?)? How does this work with custom drafts?

Reply:
16 columns is the default display for decks, 8x2